src/main/scala/uk/gov/homeoffice/drt/repository/ArrivalRepository.scala

```diff
@@ -0,0 +1,50 @@
+package uk.gov.homeoffice.drt.repository
+
+import java.time.LocalDateTime
+
+import cats.effect.{Resource, Sync}
+import skunk.codec.all._
+import skunk.implicits._
+import skunk.{Decoder, Query, Session, ~}
+
+case class ArrivalTableData(code: String,
```

Comment:
Skunk.. cool! Looks nicer than slick from what I can see here

Reply:
yes it is nice :-)